docs/src/index.md

```diff
@@ -13,15 +13,16 @@ In case you want to understand the toolbox better, we plan to offer **technical
 ## Quick start
 There are four different model types currently "fitable"
 
-1. Timeexpansion **No**, Mixed **No**  : `fit(UnfoldModel,[Any=>(f,-0.1:0.01:0.5)],evts,data_epoch)`
-1. Timeexpansion **Yes**, Mixed **No** : `fit(UnfoldModel,[Any=>(f,basisfunction)],evts,data)`
-1. Timeexpansion **No**, Mixed **Yes** : `fit(UnfoldModel,[Any=>(fLMM,-0.1:0.01:0.5)],evts,data_epoch)`
-1. Timeexpansion **Yes**, Mixed **Yes**: `fit(UnfoldModel,[Any=>(fLMM,basisfunction)],evts,data)`
+1. Timeexpansion **No**, Mixed **No**  : `fit(UnfoldModel, [Any=>(f, -0.1:0.01:0.5)], evts, data_epoch)`
+1. Timeexpansion **Yes**, Mixed **No** : `fit(UnfoldModel, [Any=>(f, basisfunction)], evts, data)`
+1. Timeexpansion **No**, Mixed **Yes** : `fit(UnfoldModel, [Any=>(fLMM, -0.1:0.01:0.5)], evts, data_epoch)`
+1. Timeexpansion **Yes**, Mixed **Yes**: `fit(UnfoldModel, [Any=>(fLMM, basisfunction)], evts, data)`
 
 With
```

Comment:
Actually, the name of the model should be written after "with"

Reply:
Why? I think the "with" refers to the code snippet which defines the formulas and basisfunction.
![grafik](https://github.com/unfoldtoolbox/Unfold.jl/assets/22366977/434dad06-79fc-4b16-9c56-8969ad6e4fe1)